app/helpers/protected_from_forgery.rb

```diff
@@ -1,18 +1,18 @@
 module ProtectedFromForgery
-  
+
   def self.included(base)
-    base.after_filter :set_csrf_cookie_for_ng
+    base.after_filter :set_xsrf_token
   end
 
   protected
-  
+
   def verified_request?
-    super || form_authenticity_token == request.headers['X_XSRF_TOKEN']
+    super || form_authenticity_token == request.headers['X-XSRF-TOKEN']
```

Comment:
Alright, I think this was our trouble here; the request stores the token in `X-XSRF-TOKEN`, rather than `X_XSRF_TOKEN`

I haven't been able to repro the above issue on clone at all after changing this; I was following along with this thingy: https://stormpath.com/blog/angular-xsrf